test_programs/plonky2_verify_failure/prove_nargo_verify_fail/VerifierTest.toml

```diff
@@ -0,0 +1,3 @@
+# This file is separate from Verifier.toml, because nargo prove overwrites Verifier.toml
+
+y = "0x0000000000000000000000000000000000000000000000000000000000000003"
```

Comment:
Perhaps worth noting what's the problem here by adding a comment: "# Note: this test should fail, because this y takes the value of x, provided in Prover.toml, rather than the value of y in Prover.toml.". Or something like this.

Reply:
Done.